classes/class-wc-connect-paypal-ec.php

```diff
@@ -221,5 +226,30 @@ public function maybe_set_reroute_requests() {
 
 			wp_safe_redirect( wc_gateway_ppec()->get_admin_setting_link() );
 		}
-    }
+
+		public function update_order_status() {
+			$orders = wc_get_orders( array( 'status' => 'on-hold' ) );
```

Comment:
What happens if there are more "on hold" orders than are included in one call to `wc_get_orders()` with default params?

Reply:
A bug 👎. `'limit' => -1` should help